uint8 action_stop=0
uint8 action_start=1

uint8 action
int64 id
int64 size
